### ISSUE TITLE
Create vm instance failed.

### ISSUE THREAD

**Issue by ihac** [zstackorg/zstack#293]
I tried to test ZStack exactly with the tutorial [_Amazon EC2 classic EIP zone_](http://zstack.org/tutorials/ec2-ui.html)(Web UI version), but it failed: **failed to start vm... libvirt error: XML error: invalid security type**. Due to my limited and initial time with IaaS software, I could not locate the fault point. Could you please figure it out? Thank you!

Here is REQUEST:
```
{
  "org.zstack.header.vm.APICreateVmInstanceMsg": {
    "name": "VM1",
    "description": null,
    "instanceOfferingUuid": "210482bc4df74f3d9fccecda27d3656a",
    "imageUuid": "2ac1f23ffbf9456dba9f4625f0c43200",
    "l3NetworkUuids": [
      "8bcbb77396504fb6ab3cc8ec076b1101"
    ],
    "rootDiskOfferingUuid": null,
    "dataDiskOfferingUuids": [],
    "zoneUuid": "ba0a0a9974e748f49b5973cb95fca22a",
    "clusterUuid": "cecb2bfe36cd4a62ab3c6bcabfcb4d07",
    "hostUuid": "e9cc51c75d924ba18fc1f32686fcd5e4",
    "resourceUuid": "49586ab4913e473694a9ca1eca7859b8",
    "defaultL3NetworkUuid": "8bcbb77396504fb6ab3cc8ec076b1101",
    "systemTags": [
      "hostname::vm1"
    ],
    "session": {
      "uuid": "d8a5f06b78254fa98f647d64e8db869c"
    }
  }
}
```

Here is the RESPONSE:
```
{
  "org.zstack.header.vm.APICreateVmInstanceEvent": {
    "success": false,
    "apiId": "6fd10a2f617f493db7916c9298491708",
    "headers": {
      "schema": {}
    },
    "error": {
      "code": "SYS.1006",
      "cause": {
        "code": "HOST.1005",
        "description": "Failed to start vm on hypervisor",
        "details": "failed to start vm[uuid:b692dd0dbfca4da9a547b9fd003dd5d9 name:virtualRouter.l3.8bcbb77396504fb6ab3cc8ec076b1101] on kvm host[uuid:e9cc51c75d924ba18fc1f32686fcd5e4, ip:10.1.11.16], because unable to start vm[uuid:b692dd0dbfca4da9a547b9fd003dd5d9, name:virtualRouter.l3.8bcbb77396504fb6ab3cc8ec076b1101], libvirt error: XML error: invalid security type"
      },
      "description": "An operation failed",
      "details": "failed to start vm[uuid:b692dd0dbfca4da9a547b9fd003dd5d9 name:virtualRouter.l3.8bcbb77396504fb6ab3cc8ec076b1101] on kvm host[uuid:e9cc51c75d924ba18fc1f32686fcd5e4, ip:10.1.11.16], because unable to start vm[uuid:b692dd0dbfca4da9a547b9fd003dd5d9, name:virtualRouter.l3.8bcbb77396504fb6ab3cc8ec076b1101], libvirt error: XML error: invalid security type"
    },
    "createdTime": 1469011965778,
    "type": {
      "_name": "key.event.API.API_EVENT"
    },
    "id": "0ffcb46a2e5745db9bf7566bde564522"
  }
}
```

**Comment by youyk** [comment]
Hi, what's your Host's operation system? 
ZStack would prefer to choose CentOS 7.2

**Comment by ihac** [comment]
Well, Ubuntu 12.04.5 LTS.
Speaking frankly, there were so many faults when installing and deploying ZStack on Ubuntu 12.04. I spent two days solving them with Google. Now it comes to the last step, Create VM Instance, and I failed to solve this fault.

I do believe that CentOS would be the best choice for ZStack. I remember that ZStack uses command `ip netns set` in some step, while the subcommand `set` only works in CentOS, not Ubuntu(WHY?)...

However, my company has no physical machine with CentOS installed...that's the key.

Anyway, I would install CentOS on my machine if I failed to handle ZStack on Ubuntu.
